sites/skeleton.dev/src/content/docs/components/dialog/svelte.mdx

```diff
@@ -0,0 +1,49 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Default from '@examples/components/dialog/svelte/default.svelte';
+import DefaultRaw from '@examples/components/dialog/svelte/default.svelte?raw';
+
+import Nested from '@examples/components/dialog/svelte/nested.svelte';
+import NestedRaw from '@examples/components/dialog/svelte/nested.svelte?raw';
+
+import Drawer from '@examples/components/dialog/svelte/drawer.svelte';
+import DrawerRaw from '@examples/components/dialog/svelte/drawer.svelte?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Default client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={DefaultRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Nested
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Nested client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={NestedRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Drawer
+
```

Comment:
Let's explain this is repurposing the Dialog modal as a side panel Drawer modal.